build.gradle.kts

```diff
@@ -91,4 +103,13 @@ kotlin {
 tasks.withType<AbstractPublishToMaven>().configureEach {
     val signingTasks = tasks.withType<Sign>()
     mustRunAfter(signingTasks)
+}
+
+// TODO: remove it after 1.9.24
```

Comment:
https://github.com/JetBrains/kotlin/releases/tag/v1.9.24 :D

Reply:
[KT-65864](https://youtrack.jetbrains.com/issue/KT-65864) K/Wasm: update Node.js to 22.x

Reply:
I'll upgrade Kotlin version and remove the line